pkg/controller/ocsinitialization/ocsinitialization_controller.go

```diff
@@ -317,3 +321,43 @@ func (r *ReconcileOCSInitialization) Reconcile(request reconcile.Request) (recon
 
 	return reconcile.Result{}, err
 }
+
+// returns a ConfigMap with default settings for rook-ceph operator
+func newRookCephOperatorConfig(namespace string) *corev1.ConfigMap {
+	var defaultCSIToleration string = `
+- key: node.ocs.openshift.io/storage
+  operator: Equal
+  value: "true"
+  effect: NoSchedule`
+
+	config := &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      rookCephOperatorConfigName,
+			Namespace: namespace,
+		},
+	}
+	data := make(map[string]string)
+	data["CSI_PROVISIONER_NODE_AFFINITY"] = "cluster.ocs.openshift.io/openshift-storage="
+	data["CSI_PROVISIONER_TOLERATIONS"] = defaultCSIToleration
+	data["CSI_PLUGIN_TOLERATIONS"] = defaultCSIToleration
+	config.Data = data
+
+	return config
+}
+
+func (r *ReconcileOCSInitialization) ensureRookCephOperatorConfig(initialData *ocsv1.OCSInitialization) error {
```

Comment:
Is this method called somewhere? I don't see where the new code is triggered.

Reply:
Forgot to add that to the commit. Fixed it.